Examples/laser_acceleration/laser_acceleration_2d_PICMI.py

```diff
@@ -0,0 +1,184 @@
+# 2D version of the LWFA example, provided for quick debugging
+# This should be the only line that needs to be changed for different codes
+# e.g. `from pywarpx import picmi`
+#      `from fbpic import picmi`
+#      `from warp import picmi`
+from fbpic import picmi
```

Comment:
I think we write this, generically at the moment:
```suggestion
from plasmacode import picmi
```